doc/clips.rst

```diff
@@ -43,25 +43,33 @@ Overview
    3   Clip 2              An image clip
    ==  ==================  ============
 
+.. _clips_cutting_slicing_ref:
+
 Cutting & Slicing
 -----------------
 OpenShot has many easy ways to adjust the start and end positions of a clip (otherwise known as cutting). The most common
 method is simply grabbing the left (or right) edge of the clip and dragging. Here is a list of methods for cutting clips in OpenShot:
 
+.. |razortool_icon| image:: ../src/images/Humanity/actions/16/edit-cut.svg
```

Comment:
```suggestion
.. |razortool_icon| image:: ../images/Humanity/actions/16/edit-cut.svg
```
The file location has changed in the repo.

Reply:
This change got missed in the update, but I can apply it here from my suggestion.